README.md

```diff
@@ -38,10 +38,12 @@ for (let workplace of workplaces) {
 
 * account
   * [`login(email, password)`](docs/account.md#loginemail-password)
+  * [`backofficeLogin()`](docs/account.md##backofficelogin)
```

Comment:
I made a compromise, I added the extra boolean argument AND kept the function since I need the call the backofficeLogin function to refresh the session.